src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/HorizontalSpacingSection.qml

```diff
@@ -62,17 +62,17 @@ Item {
     }
 
     SpinBoxPropertyView {
-        id: barWidth
+        id: measureWidth
         anchors.left: parent.horizontalCenter
         anchors.leftMargin: 4
         anchors.right: parent.right
 
-        navigationName: "Bar width"
+        navigationName: "Measure width"
         navigationPanel: root.navigationPanel
         navigationRowStart: leadingValue.navigationRowEnd + 1
 
-        titleText: qsTrc("inspector", "Bar width")
-        propertyItem: root.barWidth
+        titleText: qsTrc("inspector", "Measure width")
```

Comment:
Not sure whether this is really meant?

Reply:
I think this change is okay. This affects the USER_STRETCH property; I'm not sure that this property is meant to be exposed in the Inspector (in MU3 it was not AFAIK), since the reset button behaves a bit weirdly. Anyway, let's reports those bugs later...